openstef_dbc/utils.py

```diff
@@ -57,3 +57,34 @@ def parse_influx_result(
     result.index.name = "datetime"
     result.columns.name = ""
     return result
+
+
+def round_time_difference(time_diff, options):
+    """Rounds a time difference to the first bigger option.
+
+    Used to calculate the tAhead for forecasts"""
+    next_bigger_option = next((opt for opt in options if opt > time_diff), None)
+    if next_bigger_option is None:
+        return time_diff
+    return next_bigger_option
+
+
+max_timediff = 1000
```

Comment:
1000 does not occur in the `desired_t_aheads` list. To me it would make sense that the `max_timediff` is the same as the last item of the `desired_t_aheas` list.

Reply:
Assuming that the `desired_t_aheads` list is ordered such that the largest number in the list is the last number.

Reply:
Agreed. Also, I have added sorting of the options